src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.spec.ts

```diff
@@ -200,24 +195,6 @@ describe('DotEditPageMainComponent', () => {
         expect(nav.pageState).toEqual(mockDotRenderedPageState);
     });
 
-    it('should call reload pageSte when IframeClose evt happens', () => {
```

Comment:
Unnecessary since we're testing below when an attribute changes and with the new `pageIsSaved` flag a reload won't happen when IframeClose evt happens.

Reply:
We should tests here that nothing gets called when the dialog close without any changes.